examples/ed25519/src/ed25519.rs

```diff
@@ -0,0 +1,909 @@
+//! This crate gives a specification for the EdDSA signature scheme ed25519.
+//! As there is no proper consensus for the verification algorithm, it has been implemented in multiple ways.
+//!
+//! The different implementations and specifications differ mainly in three different ways:
```

Comment:
Can you give references here to the different specifications and which functions implements which specification?